--- makefile.orig	2023-10-24 08:54:47.000000000 -0600
+++ makefile	2023-11-26 03:58:46.000000000 -0700
@@ -164,16 +164,18 @@
 
 # if USE_NETCDF3=1, then USE_NETCDF4 must 0
 #
-# USE_NETCDF3=0
-USE_NETCDF3=1
+USE_NETCDF3=0
+# USE_NETCDF3=1
 #
-USE_NETCDF4=0
+# USE_NETCDF4=0
 # USE_NETCDF4=compile
-# USE_NETCDF4=system
+USE_NETCDF4=system
 #USE_NETCDF4=${NETCDF_INCLUDES}:${NETCDF_LIBRARIES}
+
 # USE_HDF5 is only active if USE_NETCDF4 != 0
-#USE_HDF5=0
-USE_HDF5=compile
+# For Macports, HDF5 is transitive through Netcdf, so just set to zero.
+USE_HDF5=0
+#USE_HDF5=compile
 #USE_HDF5=system
 #USE_HDF5=${HDF5_INCLUDES}:${HDF5_LIBRARIES}
 USE_REGEX=1
@@ -213,7 +215,7 @@
 lib:=${cwd}/lib
 tmp:=${cwd}/tmp
 export TMPDIR=${tmp}
-wLDFLAGS:=-L${lib}
+wLDFLAGS:=-L${lib} ${LDFLAGS}
 a:=$(shell mkdir -p ${lib})
 a:=$(shell mkdir -p ${tmp})
 wCPPFLAGS:=${CPPFLAGS} -I${cwd}/include
@@ -745,7 +747,7 @@
    jsrc=jasper-1.900.1-14ubuntu3.2.debian.tgz
    jlib=${lib}/libjasper.a
    wLDFLAGS+=-ljasper
-   wCPPFLAGS+=-I${jasperdir}/src/libjasper/include
+#   wCPPFLAGS+=-I${jasperdir}/src/libjasper/include
    a:=$(shell echo '$Hdefine USE_JASPER' >> ${CONFIG_H})
 endif
 
@@ -817,10 +819,12 @@
       wCPPFLAGS+=-I$(word 1,$a)
       wLDFLAGS+=-L$(word 2,$a)
       a:=$(shell echo "$Hdefine USE_HDF5 \"ext_lib=${USE_HDF5}\"" >> ${CONFIG_H})
+   endif
+   ifeq ($(USE_HDF5),0)
+      wLDFLAGS+=-lnetcdf
    else
-      $(error ERROR, bad USE_HDF5 in makefile")
+      wLDFLAGS+=-lnetcdf -lhdf5_hl -lhdf5 -ldl
    endif
-   wLDFLAGS+=-lnetcdf -lhdf5_hl -lhdf5 -ldl
 endif
 
 ifeq ($(USE_MYSQL),1)
@@ -899,7 +903,7 @@
 ifeq ($(USE_PNG),1)
    pngsrc=${cwd}/libpng-1.2.59.tar.gz
    pnglib=${lib}/libpng12.a
-   wLDFLAGS+=-lpng12
+   wLDFLAGS+=-lpng
    a:=$(shell echo '$Hdefine USE_PNG' >> ${CONFIG_H})
 else
    a:=$(shell echo '//$Hdefine USE_PNG' >> ${CONFIG_H})
@@ -939,10 +943,10 @@
 w=wgrib2
 prog=$w/wgrib2
 
-all:	${netcdf4src} ${hdf5src} ${prog} aux_progs/gmerge aux_progs/smallest_grib2 aux_progs/smallest_4
+all:	${prog} aux_progs/gmerge aux_progs/smallest_grib2 aux_progs/smallest_4
 
 
-${prog}:        $w/*.c $w/*.h ${jlib} ${aeclib} ${netcdf3lib} ${pnglib} ${hdf5lib} ${g2clib} ${netcdf4lib} ${iplib} ${spectrallib} ${gctpclib} ${proj4lib} ${ojlib}
+${prog}:        $w/*.c $w/*.h ${g2clib} ${iplib} ${spectrallib} ${gctpclib}
 	cd "$w" && export LDFLAGS="${wLDFLAGS}" && export CPPFLAGS="${wCPPFLAGS}" && ${MAKE}
 
 fast:        $w/*.c $w/*.h ${jlib} ${aeclib} ${netcdf3lib} ${pnglib} ${hdf5lib} ${g2clib} ${netcdf4lib} ${iplib} ${spectrallib} ${gctpclib} ${proj4lib} ${ojlib}
@@ -1021,8 +1025,10 @@
 	cd ${zdir} && export CFLAGS="${wCPPFLAGS}" && ./configure --prefix=${cwd} --static && ${MAKE} install
 	# WNE 3/2023 cd ${zdir} && export CFLAGS="" && ./configure --prefix=${cwd} --static && ${MAKE} install
 
-${g2clib}:	${jlib} ${pnglib} ${zlib}
-	cd "${g2cdir}" && export CPPFLAGS="${wCPPFLAGS}" && ${MAKE} && cp libgrib2c.a ${lib}
+${g2clib}:
+	touch ${g2clib}
+	cd "${g2cdir}" && CPPFLAGS="${wCPPFLAGS}" ${MAKE} && cp libgrib2c.a ${lib}
+	ranlib ${g2clib}
 
 ${gctpcdir}/source/makefile.gctpc:
 	cp ${gctpcsrc} tmpgctpc.tar.gz
@@ -1080,7 +1086,7 @@
 	cd "${hdf5dir}" && export CFLAGS="${hdf5CFLAGS}" && export LDFLAGS="${LDFLAGS}" && ./configure --disable-shared --with-zlib=${cwd} --disable-sharedlib-rpath --prefix=${cwd} --with-default-api-version=v110 && ${MAKE} -j 1 all check install
 
 ${iplib}:
-	cd "${ipdir}" && export FFLAGS="${wFFLAGS}" && export FTN_REAL8=${FTN_REAL8}  && ${MAKE} && cp $(notdir ${iplib}) ${iplib}
+	cd "${ipdir}" && FFLAGS="${wFFLAGS}" ${MAKE} && cp $(notdir ${iplib}) ${iplib}
 
 ${spectrallib}:
 	cd "${spectraldir}" && export FFLAGS="${wFFLAGS}" && export FTN_REAL8=${FTN_REAL8} && export FTN_LEGACY=${FTN_LEGACY} && ${MAKE} && cp $(notdir ${spectrallib}) ${spectrallib}
