--- config/FindVIGRANUMPY_DEPENDENCIES.cmake.orig	2023-07-04 05:26:03.000000000 -0600
+++ config/FindVIGRANUMPY_DEPENDENCIES.cmake	2024-05-10 01:51:27.000000000 -0600
@@ -38,7 +38,7 @@
             SET(BOOST_PYTHON_NAMES
                     boost_python-py${Python_VERSION_MAJOR}${Python_VERSION_MINOR}-mt
                     boost_python-${Python_VERSION_MAJOR}.${Python_VERSION_MINOR}-mt
-                    boost_python${Python_VERSION_MAJOR}-mt
+                    boost_python${Python_VERSION_MAJOR}${Python_VERSION_MINOR}-mt
                     boost_python-mt)
         ENDIF()
         IF(Boost_LIB_SUFFIX)
