--- cmd/gvedit/Makefile.in.orig	2025-10-19 11:08:31
+++ cmd/gvedit/Makefile.in	2025-10-31 16:16:20
@@ -123,10 +123,7 @@
 nodist_gvedit_OBJECTS = $(am__objects_1)
 gvedit_OBJECTS = $(am_gvedit_OBJECTS) $(nodist_gvedit_OBJECTS)
 am__DEPENDENCIES_1 =
-gvedit_DEPENDENCIES = $(top_builddir)/lib/gvc/libgvc.la \
-	$(top_builddir)/lib/cgraph/libcgraph.la \
-	$(top_builddir)/lib/cdt/libcdt.la \
-	$(top_builddir)/lib/util/libutil_C.la $(am__DEPENDENCIES_1) \
+gvedit_DEPENDENCIES = $(am__DEPENDENCIES_1) \
 	$(am__DEPENDENCIES_1) $(am__DEPENDENCIES_1) \
 	$(am__DEPENDENCIES_1)
 AM_V_lt = $(am__v_lt_@AM_V@)
@@ -603,10 +600,10 @@
         -DQT_USE_FAST_OPERATOR_PLUS
 
 gvedit_LDADD = \
-	$(top_builddir)/lib/gvc/libgvc.la \
-	$(top_builddir)/lib/cgraph/libcgraph.la \
-	$(top_builddir)/lib/cdt/libcdt.la \
-	$(top_builddir)/lib/util/libutil_C.la \
+	-lgvc \
+	-lcgraph \
+	-lcdt \
+	-lutil_C \
 	$(QTGUI_LIBS) \
 	$(QTCORE_LIBS) \
 	$(EXPAT_LIBS) $(Z_LIBS)
