--- m4/check_fortran.m4.orig	2024-03-20 21:50:35
+++ m4/check_fortran.m4	2025-12-02 12:16:12
@@ -495,12 +495,16 @@
 	    }
 EOF
 	doit='$CC -c ${CPPFLAGS} ${CFLAGS} conftest.c'
+        dnl AC_MSG_CHECKING( / compile conftest.c / )
 	if AC_TRY_EVAL(doit); then
 	    doit='$FC ${FFLAGS} -c conftestf.f'
+            dnl AC_MSG_CHECKING( / compile conftestf.f / )
 	    if AC_TRY_EVAL(doit); then
 	        doit='$FC -o conftest ${FFLAGS} ${FLDFLAGS} conftestf.o conftest.o ${LDFLAGS} ${LIBS}'
+	        dnl AC_MSG_CHECKING( / link stage using \$FC / )
 	        if AC_TRY_EVAL(doit); then
 		    doit=./conftest
+	            dnl AC_MSG_CHECKING( / running ./conftest / )
 		    if AC_TRY_EVAL(doit); then
 		        dnl AC_MSG_RESULT(yes)
 		        AC_MSG_RESULT(["$ctype" in C])
